### ISSUE TITLE
Stops reading when using Google UK English Male voice

### ISSUE THREAD

**Issue by iwozzy** [skechboy/SpeakIt#18]
When I use the Google UK English Male voice it stops reading the highlighted text after about (40-50) words.

This problem does not happen when using iSpeech. 

I am on Chrome  50.0.2661.87 m

**Comment by SimpleToken** [comment]
I agree. I think we all have the same issue, which is a shame because the Google voices are the best there are. Is there any TTS app that utilizes those voices properly for desktop? I tested them out with this:
https://www.audero.it/demo/speech-synthesis-api-demo.html

and they seem to be working all right. But that's not an app.
